config.go

```diff
@@ -122,6 +124,53 @@ func (h hypervisor) image() string {
 	return h.Image
 }
 
+func (h hypervisor) kernelParams() string {
+	if h.KernelParams == "" {
+		return defaultKernelParams
+	}
+
+	return h.KernelParams
+}
+
+func deserializeParams(paramString string) []vc.Param {
+	var params []vc.Param
+
+	if paramString == "" {
+		return nil
+	}
+
+	parameters := strings.Fields(paramString)
+
+	for _, param := range parameters {
+		p := strings.Split(param, "=")
+		if len(p) == 2 {
+			params = append(params, vc.Param{Key: p[0], Value: p[1]})
+		} else if len(p) == 1 {
+			params = append(params, vc.Param{Key: p[0], Value: ""})
+		}
+	}
+
+	return params
+}
+
+func serializeParams(params []vc.Param) string {
```

Comment:
I'd like to re-use such functions as much as you do. 
Do you know how should I submit PRs in both virtcontainers and runtime, and have semaphoreci/travis-ci working?